src/customerdialog.cpp

```diff
@@ -0,0 +1,259 @@
+#include "customerdialog.h"
+#include "ui_customerdialog.h"
+#include <vector>
+#include <string>
+#include <QMessageBox>
+#include <QString>
+#include <QFile>
+#include <QTextStream>
+
+using namespace std;
+
+CustomerDialog::CustomerDialog(QWidget *parent) :
+    QDialog(parent),
+    ui(new Ui::CustomerDialog)
+{
+    ui->setupUi(this);
+    /*Timer to exit thank you page and return to Welcome page
+    timer = new QTimer(this);
+    connect(timer, SIGNAL(timeout()), this, SLOT(backToWelcomePage()));
+    timer->start(20000);*/
+}
+
+//CREATING GLOBAL VARIABLES FOR STOCKS
+struct stock
+{
+    string productSerialNumber;
+    string productName;
+    double productPrice;
+    double productWeight;
+};
+
+//DECLARE VARIOUS VECTORS
+vector<stock> v;
+//vector<stock> vv;
+double totalPrice = 0.00; //for total price
+vector<stock> itemsArray;
+
+
+/*creating the inventory using array
+string productSerialNumber[5] = {"S4PYX", "9BJ2D","4QDI0", "7FGLO", "CBHM3"};
```

Comment:
remove unwanted codes that have been commented out

Reply:
I have removed these files and included the new GUI header and source files